tensilelite/Tensile/KernelWriter.py

```diff
@@ -2621,6 +2623,16 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
           module.add(self.localWriteResetOffsets(kernel, False, tensorParametersB))
 
       # tail: global read
+      # Check out VGPR for DTVA
+      vDtvResources = self.tailLoopAllocDTVVgpr(kernel, tensorParametersA, tensorParametersB)
+      for item in vDtvResources:
+        if item[0] != -1:
+          module.add(item[1])
+
+      # Check out VGPR for G2l
+      moduleMacroG2lVgpr, vgprG2L = self.tailLoopAllocG2LVgpr(kernel)
+      module.add(moduleMacroG2lVgpr)
+
```

Comment:
This is for **DTV will use the Valu VGPRs which is released in the beginning of tail loop**. @Serge45 @solaslin Please check this part of tail loop because you are engaging to Swizzle+DTV. I move the tail loop VGPR allocation for DTV here.